README.md

```diff
@@ -54,13 +51,12 @@ instead be opt-in, you can set this by adding an entry to your `package.json`
 }
 ```
 
-Scarf will now be opt-out by default, and users can set `SCARF_ANALYTICS=true`
+In constrast, Scarf will now be opt out by default, and users can set `SCARF_ANALYTICS=true`
 to opt in.
 
-Regardless of the default state, Scarf will log what it is doing to users who
-haven't explictly opted in or out.
+Without having set either true or false, and instead SCARF_ANALYTICS is set to null, Scarf will log what those users are doing.
```

Comment:
This line should be reverted, not sure what we're trying to say with this line but this statement isn't true. The purpose of this sentence is to say scarf will log its _own_ behavior to the end user.